data/json/snippets/snippets.json

```
@@ -73,6 +73,315 @@
+        "text": "It's a Dungeons & Dragons Homebrew character sheet.  This one is for a archer."
+      },
+      {
+        "id": "charsheet_46",
+        "text": "It's a Dungeons & Dragons Homebrew character sheet.  This one is for a goblin slayer."
+      },
+      {
+        "id": "charsheet_47",
+        "text": "It's a Dungeons & Dragons Homebrew character sheet.  This one is for a orc slayer."
+      },
+      {
+        "id": "charsheet_48",
+        "text": "It's a Dungeons & Dragons Homebrew character sheet.  This one is for a bounty hunter."
+      },
+      {
+        "id": "charsheet_49",
+        "text": "It's a Dungeons & Dragons Homebrew character sheet.  This one is for a prophet."
+      },
+      {
+        "id": "charsheet_50",
+        "text": "It's a Dungeons & Dragons Homebrew character sheet.  This one is for a false prophet."
+      },
+      {
+        "id": "charsheet_51",
+        "text": "It's a Dungeons & Dragons Homebrew character sheet.  This one is for a dark prophet."
+      },
+      {
+        "id": "charsheet_52",
+        "text": "It's a Dungeons & Dragons Homebrew character sheet.  This one is for a forgotten prophet."
+      },
+      {
+        "id": "charsheet_53",
+        "text": "It's a Dungeons & Dragons Homebrew character sheet.  This one is for a saint."
+      },
+      {
+        "id": "charsheet_53",
+        "text": "It's a Dungeons & Dragons Homebrew character sheet.  This one is for a dark saint."
+      },
+      {
+        "id": "charsheet_54",
+        "text": "It's a Dungeons & Dragons Homebrew character sheet.  This one is for a witch."
+      },
+      {
+        "id": "charsheet_55",
+        "text": "It's a Dungeons & Dragons Homebrew character sheet.  This one is for a elementalist."
+      },
+      {
+        "id": "charsheet_56",
+        "text": "It's a Dungeons & Dragons Homebrew character sheet.  This one is for a philosopher."
+      },
+      {
+        "id": "charsheet_57",
+        "text": "It's a Dungeons & Dragons Homebrew character sheet.  This one is for a artisan."
+      },
+      {
+        "id": "charsheet_58",
+        "text": "It's a Dungeons & Dragons Homebrew character sheet.  This one is for a builder."
+      },
+      {
+        "id": "charsheet_59",
+        "text": "It's a Dungeons & Dragons Homebrew character sheet.  This one is for a JSON miner."
+      },
+      {
+        "id": "charsheet_60",
+        "text": "It's a Dungeons & Dragons Homebrew character sheet.  This one is for a miner."
+      },
+      {
+        "id": "charsheet_61",
+        "text": "It's a Dungeons & Dragons Homebrew character sheet.  This one is for a zombie hunter."
+      },
+      {
+        "id": "charsheet_62",
+        "text": "It's a Dungeons & Dragons Homebrew character sheet.  This one is for a demonologist."
+      },
+      {
+        "id": "charsheet_63",
+        "text": "It's a Dungeons & Dragons Homebrew character sheet.  This one is for a angeologist."
```

```suggestion
        "text": "It's a Dungeons & Dragons Homebrew character sheet.  This one is for an angelologist."
```
I have learned new word 🚀